### PR TITLE
docs: add 2FA user doc

### PR DESCRIPTION
**Pull request by boojack**

### CODE REVIEW

File: content/docs/_layout.md [hunk 1]
```diff
@@ -219,6 +219,12 @@ expand_section_list: ["Introduction", "Get Started"]
 
 ---
 
+## Authentication
+
+### [2FA](/authentication/2fa)
```
Comment (tianzhou) [inline content/docs/_layout.md:224]: Let's just put it under the Administration section. Below Single Sign-On

File: content/docs/authentication/2fa.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+A time-based one-time password (TOTP) application automatically generates an authentication code that changes after a certain period of time. We recommend using cloud-based TOTP apps such as:
+
+- [Google Authenticator](https://googleauthenticator.net/)
+- [Microsoft Authenticator](https://www.microsoft.com/en-us/security/mobile-authenticator-app/)
+- [1Password](https://support.1password.com/one-time-passwords/)
+- [Authy](https://authy.com/)
+
+### Step 2 - Enable "Two-factor authentication"
+
+In the user's profile dashboard page, click the "Two-factor authentication" enable switch to enable 2FA for your account.
```
Comment (tianzhou) [inline content/docs/authentication/2fa.md:20]:
```suggestion
In the user's profile dashboard page, turn on the "Two-factor authentication" switch to enable 2FA for your account.
```

File: content/docs/authentication/2fa.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+In the "Two-factor authentication" page, you will be prompted to scan the QR code with your TOTP app. Then enter the authentication code generated by your TOTP app to verify your identity.
+
+![2fa-setup-step1](/static/docs/authentication/2fa/2fa-setup-step1.webp)
+
+If you can't scan the QR code, click **enter the text** to see a code that you can manually enter in your TOTP app instead.
+
+### Step 4 - Download the recovery codes
+
+After verified your TOTP app, you will be prompted to download your recovery codes.
```
Comment (tianzhou) [inline content/docs/authentication/2fa.md:34]:
```suggestion
After verifying your TOTP app, you will be prompted to download your recovery codes.
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/_layout.md (hunk 1 above)
Comment (boojack) [reply content/docs/_layout.md]: Done

**Comment by boojack** [comment]
PTAL

File: content/docs/_layout.md [hunk 4]
```diff
@@ -189,6 +189,8 @@ expand_section_list: ["Introduction", "Get Started"]
 
 #### [OpenID Connect (OIDC)](/administration/sso/oidc)
 
+### [Two-factor authentication](/administration/2fa)
```
Comment (tianzhou) [inline content/docs/_layout.md:192]:
```suggestion
### [Two-Factor Authentication](/administration/2fa)
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**